pubpub/platform
# fix: allow tablerow click to be triggered with keyboard and slug name to fields table

- **fix: make onclick work with keyboard navigation for table row**
- **feat: show slug name in fields table**

## Issue(s) Resolved

`onclick` for table rows was not able to be triggered through keyboard navigation.

## Test Plan

1. Go to e.g. `c/croccroc/fields`
2. Press tab a bunch until you reach the first row
3. Press either Enter or Space
4. See the modal pop up

Also try to click it to confirm the default behavior still works.

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
